test/e2e/specs/onboarding/ftme__sell.ts

```diff
@@ -25,15 +25,14 @@ import { apiCloseAccount } from '../shared';
 
 declare const browser: Browser;
 
-describe( DataHelper.createSuiteTitle( 'FTME: Sell' ), function () {
```

Comment:
**question (non-blocking):** (clarification) Is there a particular reason for deleting `DataHelper.createSuiteTitle())`? Are we deprecating this?